ui/src/app/Activation/activation-stdout.tsx

```diff
@@ -8,12 +9,49 @@ import {
   StackItem,
 } from '@patternfly/react-core';
 import {useIntl} from "react-intl";
+import {getServer} from "@app/utils/utils";
 import {renderActivationTabs} from "@app/Activation/Activation";
 import Ansi from "ansi-to-react";
 import {ActivationType} from "@app/Activations/Activations";
 
-const ActivationStdout: React.FunctionComponent<{activation: ActivationType, stdout: string[]}> = ({activation, stdout}) => {
+const endpoint3 = 'http://' + getServer() + '/api/activation_instance_logs/?activation_instance_id=';
```

Comment:
Can you please use better naming instead of `endpoint1, endpoint2, endpoint3`?

Reply:
@cutwater - I will refactor all the endpoints in a separate PR - it's on my ToDo list

Reply:
Ok